laws/ord.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+const {lte, equals} = require('..');
+
+/**
+   ### Ord
+
+   1. `a.lte(b)` or `b.lte(a)` (totality)
+   2. If `a.lte(b)` and `b.lte(a)`, then `a.equals(b)` (antisymmetry)
+   3. If `a.lte(b)` and `b.lte(c)`, then `a.lte(c)` (transitivity)
+
+**/
+
+const totality = eq => f => g => {
+  const a = f[lte](g);
+  const b = g[lte](f);
+  const c = true;
+  return eq(a || b, c);
+};
```

Comment:
I think all of the laws are like this—`equals` included. I can't remember why, but there was a reason we switched to this way of writing the laws.